projects/addon-doc/components/demo/demo.style.less

```diff
@@ -177,23 +180,41 @@
     }
 }
 
-.t-group,
 .t-checkbox {
     margin-left: 0.75rem;
 }
 
 .t-button,
+.t-copy-code,
 .t-form-controls {
     margin-top: 0.75rem;
 }
 
+.t-code-container {
+    background: var(--tui-base-01);
+    border-radius: 0 0 var(--tui-radius-m) var(--tui-radius-m);
+}
+
+.t-code {
+    position: relative;
+    z-index: 1;
+    overflow: hidden;
+    height: 100%;
+    --tui-code-action-top: 1.5rem;
+}
+
 @supports (position: sticky) {
     @media screen and (min-height: 37.5rem) {
         :host {
-            &._sticky {
+            &._sticky._allow_sticky {
                 position: sticky;
-                top: 4.625rem;
+                top: 0;
                 z-index: 2;
+
+                > .t-wrapper {
```

Comment:
I don't think this is needed, I have disabled those styles and everything still works as expected.

Reply:
fixed